### ISSUE TITLE
What is the warmup stage doing exactly? 

### ISSUE THREAD

**Issue by yashkant** [facebookresearch/banmo#11]
Hi, thanks for releasing the code! 

I attempted training on the human video, and saw that nerf_coarse is trained for 5 epochs to warmup using an SMPL mesh loaded from `mesh_material` folder.

However,  when I visualized it from `tmp/smpl_27554.obj` in blender it just seemed to be an ellipsoid [image below], is this expected? 

Appreciate your thoughts, thanks! 

![image](https://user-images.githubusercontent.com/17530895/164049315-c21fc817-5459-4016-a852-f92b4c335f91.png)

**Comment by gengshan-y** [comment]
Yes! It is supposed to be a ellipsoid instead of SMPL shape. As mentioned in "Optimization details" of the [preprint](https://banmo-www.github.io/banmo-2-14.pdf) -- "We initialize MLP_SDF such that it approximates a unit sphere [65]." 

If you open up `logdir/$seqname-e120-b256-init/mesh_rest-00.obj` to look at the initial mesh at iteration 0, you'll find the following mesh
![image](https://user-images.githubusercontent.com/13134872/164127184-40eddd25-d715-4557-aa16-b7ca8d3c5e16.png)

Note it is a surface extracted from the warmup-ed MLP_SDF by running marching cubes.

**Comment by yashkant** [comment]
Thanks for the prompt clarification! 

I was a bit surprised to find an ellipsoid there and wanted to confirm in case I had introduced a bug somewhere.

**Comment by zhan-xu** [comment]
Hi! When I tried to train on the human video, it reported "No such file or directory: 'xxx/banmo/tmp/smpl_27554.obj'". Could you let me know how you got this small_27554.obj? Thank you! @yashkant

**Comment by gengshan-y** [comment]
Hi, where did it report this error? This smpl_27554.obj file is created [here](https://github.com/facebookresearch/banmo/blob/26156475e2c464141c1e9a810ca265e6eb976afe/nnutils/banmo.py#L394-L398) for visualization.

**Comment by zhan-xu** [comment]
Hi @gengshan-y thanks for the quick reply. I ran the following command
`seqname=adult7
python preprocess/img2lines.py --seqname $seqname
bash scripts/template-accu.sh 0 $seqname 10001 "" ""
`

`scripts/template.sh` gave the same error.

**Comment by gengshan-y** [comment]
Could you attach an error log? Is it possible there is no `tmp/` dir so that it is not able to save files to it?

**Comment by zhan-xu** [comment]
Thanks @gengshan-y. That is exactly the problem. The error is gone when I first 'mkdir tmp'

I have another problem. When I tried the pertained human model, I first downloaded human-cap.npy and human-cap.pth, and put them into the tmp folder (as well as opts.log?). I then ran the following commands:
```
seqname=adult7
bash scripts/render_mgpu.sh 0 $seqname tmp/human-cap.pth "0 1 2 3 4 5 6 7 8 9" 256
```
Is this the right way? I got however the output as attached, which seems not right. Could you please take a look?
[adult7_pretrain_output.txt](https://github.com/facebookresearch/banmo/files/8942098/adult7_pretrain_output.txt)

**Comment by gengshan-y** [comment]
The problem might be the inconsistent flags stored in the opts.log file. You want to change `--nouse_human` to `--use_human`. Let me know if there is still a problem.
